cmake/InstallMSYS2.cmake

```diff
@@ -82,6 +83,7 @@ if(WIN32)
             ERROR_QUIET
             OUTPUT_STRIP_TRAILING_WHITESPACE
         )
+        string(REPLACE "\\" "/" MLT_DIRECTORY "${MLT_DIRECTORY}")

```

Comment:
You can use `cygpath -m <path>` and get Windows paths with slashes, and you won't need to replace them..